app/frontend/utils/formio-helpers.ts

```diff
@@ -0,0 +1,16 @@
+import { IFormIORequirement, ISingleRequirementFormJson } from "../types/types"
+
+export const singleRequirementFormJson = (requirementJson: IFormIORequirement): ISingleRequirementFormJson => {
+  requirementJson.conditional = null
+  requirementJson.customConditional = null
+
+  return {
+    id: requirementJson.id,
+    key: `single-${requirementJson.key}`,
+    components: [requirementJson],
+  }
+}
+
+export const isFieldSetKey = (key: string) => {
+  return key.split("|").length === 4
```

Comment:
Not sure of the exact use case, but this might be a bit flaky, as if anyone introduces new  key with the same split length it will break. If there is a more deterministic way, I would prefer using that, otherwise this is is fine

Reply:
the form is just like this, I don't think there is a better way